Resources/Prototypes/Entities/Objects/Weapons/Throwable/bola.yml

```diff
@@ -35,16 +55,18 @@
           collection: MetalBreak
       - !type:DoActsBehavior
         acts: [ "Destruction" ]
-  - type: DamageOnLand
-    damage:
-      types:
-        Blunt: 5
-  - type: Ensnaring
-    freeTime: 2.0
-    breakoutTime: 3.5 #all bola should generally be fast to remove
-    walkSpeed: 0.7 #makeshift bola shouldn't slow too much
-    sprintSpeed: 0.7
-    staminaDamage: 55 # Sudden weight increase sapping stamina
-    canThrowTrigger: true
-    canMoveBreakout: true
 
+  - type: entity
+    name: energy bola
+    id: BolaEnergy
+    parent: [BaseBola]
+    description: An advanced hardlight criminal entangling tool. Otherwise known as an expensive piece of string.
+    components:
+    - type: Sprite
+      sprite: Objects/Weapons/Throwable/energy_bola.rsi
+    - type: DamageOnLand
+      damage:
+        types:
+        Heat: 5
+    - type: Ensnaring
+      destroyOnRemove: true
```

Comment:
```suggestion
- type: entity
  name: energy bola
  id: BolaEnergy
  parent: BaseBola
  description: An advanced hardlight criminal entangling tool. Otherwise known as an expensive piece of string.
  components:
  - type: Sprite
    sprite: Objects/Weapons/Throwable/energy_bola.rsi
  - type: DamageOnLand
    damage:
      types:
      Heat: 5
  - type: Ensnaring
    destroyOnRemove: true
```